tools/tests/test_osqueryd.py

```diff
@@ -93,5 +93,38 @@ def test_5_daemon_sigint(self):
         acceptable_retcodes = [-1, -2, -1 * signal.SIGINT]
         self.assertTrue(daemon.retcode in acceptable_retcodes)
 
+    def test_6_logger_mode(self):
+        logger_path = os.path.join(test_base.CONFIG_DIR, "logger-mode-tests")
```

Comment:
Right! So this is testing the status logs: INFO, WARNING, ERROR emitted by glog. These are not the same as the results logging which are the added/removed/etc to a default `./osqueryd.results.log` which is what the `FilesystemLoggerPlugin` is writing within `logString`. To get the result log to produce some data we could do either:

A) Have the `FilesystemLoggerPlugin` create the intended log file without any bytes on startup/construction. 
B) Add a small config to this integration test that scheduled a "select * from time" with an interval=1 and wait until the log is created.

A takes a bit more code, B is a little janky will bloat the python tests by at least another 1s, but is easy and fast!